food-waste-express-api/__tests__/sortProductsByDropdownMenu.js

```diff
@@ -0,0 +1,32 @@
+const axios = require('axios').default;
+
+
+describe("Fetch product in sorted order by dropdown menu", () => {
+        test('SortProductTest', () => {
```

Comment:
What is the difference between each of these 'test('SortProductTest',' ? Can we label these better or reduce it to just 1 if they are similar?

Reply:
I see, seems like these are different sort methods. 
1. label these something like sortProductListByDistanceHighToLowTest
2. Checking if the return length is not as valuable of a test as checking to make sure that the ordering is correct. In the case of sortProductListByDistanceHighToLowTest, make sure to loop through the products to ensure the distance is actually set from high to low etc.